src/pages/home/components/events/Events.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+import { EventLayout } from "shared/layout/EventLayout";
+import { UPCOMINGEVENTS, RANTANGAN } from "./events_data";
+
+export const Events = () => {
+  return (
+    <div className="flex justify-between flex-wrap">
+      <EventLayout
```

Comment:
I think it's better to use `constants.map((constant) => { <EventLayout />}`. So you also need to change the data file to a list of objects.

Reply:
Ahh I see where you are coming from. I will take this into consideration when I work on the responsive ticket. I'll see which one is a better approach.